### PR TITLE
Typo Fixed in code.

### PR DESCRIPTION
**Pull request by realadityayadav**
### Fixed Issues (if relevant)

1. Fixed Typo of $extensionAttrbutes -> $extensionAttributes
    FILE - app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php
2. Fixed Typo of WigetTabs -> WidgetTabs
    FILE - app/code/Magento/Catalog/Block/Adminhtml/Product/Edit/Tabs.php

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @adityayadav-cedcoss. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=20059) <br/>All committers have signed the CLA.

**Review by nuzil — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @nuzil, thank you for the review.
[ENGCOM-3785](https://jira.corp.magento.com/browse/ENGCOM-3785) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@adityayadav-cedcoss thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php [hunk 1]
```diff
@@ -180,7 +180,7 @@ class ProductTest extends \PHPUnit\Framework\TestCase
     /**
      * @var \PHPUnit_Framework_MockObject_MockObject
      */
-    private $extensionAttrbutes;
+    private $extensionAttributes;
```
Comment (nuzil) [inline app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php:183]: Can you please check Unit Tests, such variable is already presented in codebase

**Review by nuzil — CHANGES_REQUESTED** [review]

**Comment by realadityayadav** [comment]
Sure @nuzil I am on it.

**Comment by nuzil** [comment]
@adityayadav-cedcoss were those tests you changed broken before? Or how this change is related to Typo's fix?

**Comment by realadityayadav** [comment]
@nuzil There were typos that I have found and addressed but the test failed due to duplication of the variable name in my first commit. 
The 2nd set of changes that I committed fixed the duplication error as well as typo's.

File: app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php [hunk 2]
```diff
@@ -557,14 +558,23 @@ public function testGetCategory()
...
     }
 
+    public function testGetCategoryIdWhenProductNotInCurrentCategory()
+    {
+        $this->model->setData('category_ids', [12]);
+        $this->category->expects($this->once())->method('getId')->will($this->returnValue(10));
+        $this->registry->expects($this->any())->method('registry')->will($this->returnValue($this->category));
+        $this->assertFalse($this->model->getCategoryId());
+    }
+
```
Comment (p-bystritsky) [inline app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php]: This test is very strange as it's failing initially.

**Review by p-bystritsky — CHANGES_REQUESTED** [review]
@adityayadav-cedcoss, please edit testGetCategoryIdWhenProductNotInCurrentCategory test. Now it's failing.

File: app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php (hunk 2 above)
Comment (milindsingh) [reply app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php]: Is it relevant to this PR?
I don't think the changes in this cause to fail the new case.

Comment (p-bystritsky) [reply app/code/Magento/Catalog/Test/Unit/Model/ProductTest.php]: @milindsingh anyway, any newly added test should pass successfully.

**Pull request merged**